packages/apputils-extension/src/index.ts

```diff
@@ -650,7 +650,7 @@ const utilityCommands: JupyterFrontEndPlugin<void> = {
     });
 
     commands.addCommand(CommandIDs.displayShortcuts, {
-      label: trans.__('Show Keyboard Shortcuts'),
+      label: trans.__('Show Keyboard Shortcuts...'),
```

Comment:
Please use the ellipsis character …, which looks very similar to three periods, but which is not the same.

* Ellipsis: `…`
* Three periods: `...`

Reply:
Thanks, i can redo the changes, they're small to fix this.